release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -205,6 +205,10 @@ For more information, see xref:../networking/external_dns_operator/nw-installing
 
 SR-IOV support is now available for xref:../networking/hardware_networks/about-sriov.adoc#supported-devices_about-sriov[Pensando DSC cards]. OpenShift SR-IOV is supported, but you must set a static, Virtual Function (VF) media access control (MAC) address using the SR-IOV CNI config file when using SR-IOV.
 
+[id="ocp-4-11-ovn-kubernetes-ipsec-enable"]
+==== OVN-Kubernetes network provider: Enable IPsec at runtime
+If you are using the OVN-Kubernetes cluster network provider, you can now enable IPsec encryption after cluster installation. For more information on how to enable IPsec, see xref:../networking/ovn_kubernetes_network_provider/configuring-ipsec-ovn.adoc#configuring-ipsec-ovn[Configuring IPsec encryption].
```

Comment:
Change to "For more information about how..." RH SG. Not explicitly mentioned, but it is shown this way ("about" instead of "on").